RubberduckTests/Inspections/UnreachableCaseInspectionTests.cs

```diff
@@ -0,0 +1,2398 @@
+        {
+            TestBinaryOp(MathSymbols.EXPONENT, operands, expected, typeName);
+        }
+
+        [TestCase("10_Mod_3", "1", "Currency")]
+        [Category("Inspections")]
+        public void UciUnit_Modulo(string operands, string expected, string typeName)
+        {
+            TestBinaryOp(MathSymbols.MODULO, operands, expected, typeName);
+        }
+
+        [TestCase("10_=_3", "False")]
+        [TestCase("10_=_10", "True")]
+        [TestCase("10_<>_3", "True")]
+        [TestCase("10_<>_10", "False")]
+        [TestCase("10_<_3", "False")]
+        [TestCase("10_<=_10", "True")]
+        [TestCase("10_>_11", "False")]
+        [TestCase("10_>=_10", "True")]
+        [TestCase("5_And_0", "False")]
+        [TestCase("5_Or_0", "True")]
+        [TestCase("5_Xor_6", "False")]
+        [TestCase("6.5_>_5.2", "True")]
+        [Category("Inspections")]
+        public void UciUnit_LogicBinaryConstants(string operands, string expected)
+        {
+            GetBinaryOpValues(operands, out IParseTreeValue LHS, out IParseTreeValue RHS, out string opSymbol);
+
+            var result = Calculator.Evaluate(LHS, RHS, opSymbol);
+
+            Assert.AreEqual(expected, result.ValueText);
+            Assert.IsTrue(result.ParsesToConstantValue);
+        }
+
+        //        Dim A, B, C, D, MyCheck
+        //A = 10: B = 8: C = 6: D = Null    ' Initialize variables.
+        //MyCheck = A > B Eqv B > C    ' Returns True.
+        //MyCheck = B > A Eqv B > C    ' Returns False.
+        //MyCheck = A > B Eqv B > D    ' Returns Null.
+        //MyCheck = A Eqv B    ' Returns -3 (bitwise comparison).
+        [TestCase("True_Eqv_True", "True")]
+        [TestCase("False_Eqv_True", "False")]
+        [TestCase("True_Eqv_False", "False")]
+        [TestCase("False_Eqv_False", "True")]
+        [Category("Inspections")]
+        public void UciUnit_LogicEqvOperator(string operands, string expected)
+        {
+            GetBinaryOpValues(operands, out IParseTreeValue LHS, out IParseTreeValue RHS, out string opSymbol);
+
+            var result = Calculator.Evaluate(LHS, RHS, opSymbol);
+
+            Assert.AreEqual(expected, result.ValueText);
+            Assert.IsTrue(result.ParsesToConstantValue);
+        }
+
+        [TestCase("True_Imp_True", "True")]
+        [TestCase("False_Imp_True", "True")]
+        [TestCase("True_Imp_False", "False")]
+        [TestCase("False_Imp_False", "True")]
+        [Category("Inspections")]
+        public void UciUnit_LogicImpOperator(string operands, string expected)
+        {
+            GetBinaryOpValues(operands, out IParseTreeValue LHS, out IParseTreeValue RHS, out string opSymbol);
+
+            var result = Calculator.Evaluate(LHS, RHS, opSymbol);
+
+            Assert.AreEqual(expected, result.ValueText);
+            Assert.IsTrue(result.ParsesToConstantValue);
+        }
+
+        [TestCase("Not_False", "True")]
+        [TestCase("Not_True", "False")]
```

Comment:
Do we need a test case for `Not 1` evaluating to `True`?

Reply:
I can add certainly one...but I think the correct result of Not 1 is `False` ?

Reply:
Then, we definitely needed the test case.  Done.